src/python/WMCore/Storage/DeleteMgr.py

```diff
@@ -84,43 +75,52 @@ def initialiseSiteConf(self):
         Extract required information from site conf and TFC
 
         """
-
-        implName = self.siteCfg.localStageOut.get("command", None)
-        if implName == None:
-            msg = "Unable to retrieve local stage out command\n"
-            msg += "From site config file.\n"
-            msg += "Unable to perform StageOut operation"
-            raise StageOutInitError(msg)
-        msg = "Local Stage Out Implementation to be used is:"
-        msg += "%s\n" % implName
-
-        pnn = self.siteCfg.localStageOut.get("phedex-node", None)
-        if pnn == None:
-            msg = "Unable to retrieve local stage out phedex-node\n"
-            msg += "From site config file.\n"
-            msg += "Unable to perform StageOut operation"
-            raise StageOutInitError(msg)
-        msg += "Local Stage Out PNN to be used is %s\n" % pnn
-        catalog = self.siteCfg.localStageOut.get("catalog", None)
-        if catalog == None:
-            msg = "Unable to retrieve local stage out catalog\n"
-            msg += "From site config file.\n"
-            msg += "Unable to perform StageOut operation"
-            raise StageOutInitError(msg)
-        msg += "Local Stage Out Catalog to be used is %s\n" % catalog
-
-        try:
-            self.tfc = self.siteCfg.trivialFileCatalog()
-            msg += "Trivial File Catalog has been loaded:\n"
-            msg += str(self.tfc)
-        except Exception as ex:
-            msg = "Unable to load Trivial File Catalog:\n"
-            msg += "Local stage out will not be attempted\n"
-            msg += str(ex)
+        
+        self.stageOuts = self.siteCfg.stageOuts
+
+        msg = "There are %s stage out definitions.\n" % len(self.stageOuts)
```

Comment:
As far as I can see, this log line, line 102 and maybe a few more will only be logged if we have an error or exception. Can you please revisit it and also print things out in info mode?

Reply:
Done. Use self.logger.info